### DISCUSSION TITLE
I bought a Raspberry Pi bush

### DISCUSSION THREAD

**Discussion by Kristopian** [diasurgical/DevilutionX#3941] [Q&A] [answered]
Sooooo... I splurged a bit and decided to order some Raspberry Pi, specifically 4 Raspberry Pi model B with 8GB, 4 Raspberry Pi model B with 2GB, 10 Raspberry Pi Pico, and a Raspberry Pi 400 kit. I am wondering if DevilutionX is already on Raspberry Pi OS, or if any tools exist to help port DevilutionX to Raspberry Pi OS, if not, then time to add a new build.

OOF, I just noticed #3838

**Comment by AJenbo** [comment, marked as answer]
We provide a [32bit image for Raspberry Pi](https://github.com/diasurgical/devilutionX/releases/download/1.3.0/devilutionx-linux-armhf.zip), if you want to have a pure 64bit system then you need to build it your self for now.

**Reply by Kristopian** [reply]
Got it

**Reply by qndel** [reply]
> Sooooo... I splurged a bit and decided to order some Raspberry Pi, specifically 4 Raspberry Pi model B with 8GB, 4 Raspberry Pi model B with 2GB, 10 Raspberry Pi Pico, and a Raspberry Pi 400 kit. I am wondering if DevilutionX is already on Raspberry Pi OS, or if any tools exist to help port DevilutionX to Raspberry Pi OS, if not, then time to add a new build.
> 
> OOF, I just noticed #3838

Why would you get so many lol

**Reply by Kristopian** [reply]
@qndel I just thought I could have some extra in case I did something else that needed raw computing power(like Dwarf Fortress), or just wanted to make something like a slightly intelligent robot.

**Reply by StephenCWills** [reply]
My guess was a mini-warehouse for gear-farming bots in DevilutionX.

**Reply by Kristopian** [reply]
> My guess was a mini-warehouse for gear-farming bots in DevilutionX.

I have an idea, EXP farming bots, for when power leveling is too much and you don't feel like cheating.

PS
Quote reply is strange

PSS
Wrong image

**Reply by Kristopian** [reply]
My Pis just came in, and now I have a Raspberry Pi model B with 8GB RAM, a Delkin Devices 64GB (DDMSDG200064)(and yes I know this is overkill) running Diabl OS, which is a heavily modified Raspberry Pi OS that only has the necessary resources to play DevilutionX, and nothing else.

**Reply by AJenbo** [reply]
At first I thought 700MHz CPU was a bit low spec, but then I realized that with 8GB RAM you probably mean some later model :)

**Reply by Kristopian** [reply]
> At first I thought 700MHz CPU was a bit low spec, but then I realized that with 8GB RAM you probably mean some later model :)

The Raspberry Pi 4 model B has a 1.5GHz CPU which can be overclocked to 1.8GHz with a slight feeling of uneasiness, while the newer Raspberry Pi 4 model B (I have 3 of these, only with 8GB RAM) can be overclocked to 1.8GHz which is the same as the 400 without worrying in the slightest because of the enhanced specs that they silently added to more recently produced model Bs.

**Reply by AJenbo** [reply]
you left out that it was the 4th gan. The Raspberry Pi model B has a 700Mhz cpu and 512MB RAM

**Reply by Kristopian** [reply]
> you left out that it was the 4th gan. The Raspberry Pi model B has a 700Mhz cpu and 512MB RAM

Wow, I missed the fact that they were 4s, even though I tried to list the specs. *Dies inside*
